uber/config.py

```diff
@@ -195,6 +195,11 @@ def PAGE(self):
         return cherrypy.request.path_info.split('/')[-1]
 
     @property
```

Comment:
It might have just been swallowing the exception in the template, since that's what Django does by default.

Also, one note about ``to_dict`` is that by default it only returns database columns, so you won't get properties such as ``full_name`` unless you explicitly ask for it, e.g. you could say ``.to_dict(['full_name'])`` but that would ONLY return the ``full_name`` field instead of everything else AND the ``full_name`` field... though at this point you only really need the full name field, so that mgiht be okay.